lib/store/index.js

```diff
@@ -124,6 +127,35 @@ export class Store {
     this.startingKernels.set(kernelDisplayName, true);
   }
 
+  addFileDisposer(editor: atom$TextEditor, filePath: string) {
+    const fileDisposer = new CompositeDisposable();
+
+    if (isUnsavedFilePath(filePath)) {
+      fileDisposer.add(
+        editor.onDidSave(event => {
+          fileDisposer.dispose();
+          this.addFileDisposer(editor, event.path); // Add another `fileDisposer` once it's saved
+        })
+      );
+      fileDisposer.add(
+        editor.onDidDestroy(() => {
+          this.kernelMapping.delete(filePath);
+          fileDisposer.dispose();
+        })
+      );
+    } else {
+      const file: atom$File = new File(filePath);
```

Comment:
Is this the atom `File` or the [ browser API ](https://developer.mozilla.org/en-US/docs/Web/API/File)

If it's atom's does it need to be imported, or are they the same class?

Reply:
I see the import now 😄

Reply:
Yeah, I'm using Atom API and it's imported.